### DISCUSSION TITLE
Package.json

### DISCUSSION THREAD

**Discussion by ElectronicsArchiver** [pulsar-edit/.github#101] [Pulsar Support] [answered]
Is there an existing, up-to-date resource on what fields
Pulsar package manifests can contain, looking at old
packages I'm not quite sure what comes from Atom
and what from other tools, etc ..

**Comment by confused-Techie** [comment, marked as answer]
Our [docs](https://pulsar-edit.dev/docs/launch-manual/sections/core-hacking/#package-json) on this subject are fully up to date. The keys available in a `package.json` haven't been changed since the fork from Atom.

**Reply by ElectronicsArchiver** [reply]
Thank you, couldn't seem to find that ..

**Reply by ElectronicsArchiver** [reply]
I think there are some mistakes in the documentation:

- Docs say field is called `styles` however, the code
  specifies `styleSheets`

- Undocumented `mainStylesheet` field

https://github.com/pulsar-edit/pulsar/blob/3573c348e7af38a7bd343382e53efbb95915f72d/src/package.js#L623

**Reply by confused-Techie** [reply]
@ElectronicsArchiver thanks for the info, I'll make an issue over on our docs repo, thanks for pointing this out!
